contracts/zero-ex/contracts/src/migrations/InitialMigration.sol

```diff
@@ -47,8 +53,13 @@ contract InitialMigration {
     ///      transfers ownership to `owner`, then self-destructs.
     ///      Only callable by `deployer` set in the contstructor.
     /// @param owner The owner of the contract.
+    /// @param features Features to bootstrap into the proxy.
     /// @return zeroEx The deployed and configured `ZeroEx` contract.
-    function deploy(address payable owner) public virtual returns (ZeroEx zeroEx) {
+    function deploy(address payable owner, BootstrapFeatures memory features)
```

Comment:
So now the migrator does not deploy the feature contracts for you. Turns out some of these are quite large.